### PR TITLE
Use Firefox to generate pdf report

### PR DESCRIPTION
**Pull request by Yash12Bansal**
Now "Discharge Summary" button shows PDF preview both on Chrome and Firefox
Here is the link to the issue-
https://github.com/coronasafe/care_fe/issues/5611

### CODE REVIEW

File: package-lock.json
Comment (rithviknishad) [inline package-lock.json:1]: Why is this tracked?

File: src/Components/Facility/DischargeSummaryModal.tsx [hunk 1]
```diff
@@ -42,7 +42,15 @@ export default function DischargeSummaryModal(props: Props) {
...
-        window.open(res.data.read_signed_url, "_blank");
+        const blob = new Blob([res.data], { type: "application/pdf" });
+        const url = URL.createObjectURL(blob);
+        const isFirefox =
+          navigator.userAgent.toLowerCase().indexOf("firefox") > -1;
+        if (isFirefox) {
+          window.open(url, "_blank");
+        } else {
+          window.open(res.data.read_signed_url, "_blank");
+        }
```
Comment (rithviknishad) [inline src/Components/Facility/DischargeSummaryModal.tsx:53]: We can use this same for all browsers, right?

Comment (Yash12Bansal) [reply src/Components/Facility/DischargeSummaryModal.tsx:53]: yes

Comment (Yash12Bansal) [reply src/Components/Facility/DischargeSummaryModal.tsx:53]: yes

Comment (Yash12Bansal) [reply src/Components/Facility/DischargeSummaryModal.tsx:53]: what changes are needed ? you have requested one change

Comment (rithviknishad) [reply src/Components/Facility/DischargeSummaryModal.tsx:53]: Do the same for all browsers then. Keep the code simple.

Untrack your `package.json` and `package-lock.json` changes. 
Ensure you've proper line endings configured in your git.

**Comment by Yash12Bansal** [comment]
I had made the changes but forgot to make a pr as I was working on other issues. So, sorry for that delay
I have made all the changes and it now supports all major browsers. I have also untracked package-lock.json files
So please review and accept the pr
https://github.com/coronasafe/care_fe/pull/5682

**Comment by rithviknishad** [comment]
You could've pushed the changes in this PR itself on the same branch right?

**Pull request closed without merging**